### PR TITLE
Thinkdocs 80 remove all string concats

### PR DESCRIPTION
**Pull request by arwehrman**
This PR does the following:

- removes outdated string concatenate from Javascript section 
- adds template literals

Closes: [THINKDOCS-80](https://thinkcompany.atlassian.net/browse/THINKDOCS-80?atlOrigin=eyJpIjoiZDliOTZlMGFhMWNkNDdkOGEyMTMyMGIzNmJkOGM0YmEiLCJwIjoiaiJ9)

### CODE REVIEW

File: content/javascript/general-authoring/index.md [hunk 1]
```diff
@@ -241,7 +242,7 @@ function inbox(messages) {
 
   for (i = 0; i < length; i++) {
       // use direct assignment in this case because we're micro-optimizing.
-      items[i] = '<li>' + messages[i].message + '</li>';
+      items[i] = `<li>${messages[i].message}</li>`;
   }
 
   return '<ul>' + items.join('') + '</ul>';
```
Comment (brendanmckeown) [inline content/javascript/general-authoring/index.md]:
```suggestion
  return `<ul>{items.join('')}</ul>`;
```

**Review by brendanmckeown — APPROVED** [review]

File: content/javascript/general-authoring/index.md [hunk 2]
```diff
@@ -171,47 +171,48 @@ Use single quotes `''` for strings.
...
-var name = "Bob Parr";
+const name = "Bob Parr";
 
-// good
-var name = 'Bob Parr';
+const fullName = "Bob " + this.lastName;
 
-// bad
-var fullName = "Bob " + this.lastName;
+var errorMessage = 'This is a super long error that was thrown because ' +
```
Comment (think-john) [inline content/javascript/general-authoring/index.md]: Is the goal of this PR "kill all `var` s"? If so, this one escaped.

**Review by think-john — CHANGES_REQUESTED** [review]
I suspect that one `var` escaped the conflagration. Find it and destroy it 

image

File: content/javascript/general-authoring/index.md (hunk 2 above)
Comment (arwehrman) [reply content/javascript/general-authoring/index.md]: It wasn't, that was another PR but in any case I'll remove it. :)

**Review by andyknapp — APPROVED** [review]
These changes look good to me Mandy!

**Review by think-john — APPROVED** [review]
🚢 SHIP IT 🚢

**Pull request merged**